docs/overview/vision-of-the-thousand-brains-project/challenging-preconceptions.md

```diff
@@ -11,4 +11,4 @@ Below, we highlight some of the most important differences between the system we
 
 - Despite the previous point, each modeling system can learn complete models of objects and recognize them on its own. A single modeling unit should be able to perform all basic tasks of object recognition and manipulation. Using more modeling units makes the system faster and more efficient, and supports compositional and abstract representations, but a single learning module is itself a powerful system. In the single model scenario, inference always requires movement to collect a series of observations, in the same way that recognizing a coffee cup with one of your fingers requires moving across its surface.
 
-- All models are structured by reference frames. An object is not just a bag of features. It is a collection of features at locations. The relative locations of features to each other is more important than the features themselves.
\ No newline at end of file
+- All models are structured by reference frames. An object is not just a bag of features. It is a collection of features at locations. The locations of features, relative to each other and to the observer, are more important than the features themselves.
```

Comment:
Models don't store locations relative to the observer. They only store features relative to each other (-> object specific reference frame) This is required to recognize the object independent of where it is relative to the observer.